pkgs/desktops/gnome/core/gnome-desktop/default.nix

```diff
@@ -75,6 +75,8 @@ stdenv.mkDerivation rec {
   mesonFlags = [
     "-Dgtk_doc=true"
     "-Ddesktop_docs=false"
+    "-Ddate_in_gnome_version=false"
+    "-Dgnome_distributor=NixOS"
```

Comment:
Does this option apply to other packages too?

Reply:
grepping through the openSUSE packages only gnome-desktop has it https://github.com/bmwiedemann/openSUSE

Reply:
This is used for XML file serving as a data source to About panel of GNOME Control Center, displaying info about system. That is only relevant for gnome-desktop which contains the file.